app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/model/Event.kt

```diff
@@ -34,6 +35,18 @@ data class Event(@PrimaryKey
         const val defaultDuration = 7200000 // Milliseconds
     }
 
+    fun getFormattedDateTime(): String {
+        return "${getFormattedDate()}, ${getFormattedTime()}"
+    }
+
+    fun getFormattedDate(): String {
+        return DateTimeFormat.forPattern("dd MMMM yyyy").print(start)
+    }
+
+    fun getFormattedTime(): String {
+        return DateTimeFormat.forPattern("HH:mm").print(start)
```

Comment:
return DateTimeFormat.shortTime().print(start)

This will respect the current locale, e.g. "2:00 pm" vs "14:00"

Reply:
see answer above this post. shortTime will return "2:00 nachm." instead of "14:00"